### DISCUSSION TITLE
Rename fails (not sure if bug)

### DISCUSSION THREAD

**Discussion by rieje** [jarun/nnn#1693] [Q&A] [answered]
Selecting multiple files and pressing `r`, it just says `failed` at the bottom. I can only rename one file at a time using `R`. How to go about debugging this?

**Comment by N-R-K** [comment, marked as answer]
A. Did you edit `O_NOBATCH` when compiling `nnn`?
B. Is `.nmv` plugin present on your plugin directory?

**Reply by rieje** [reply]
Adding the `.nmv` plugin solved this for me but I thought batch rename was builtin and I don't recall using any plugins (99% sure). Thanks.

**Reply by N-R-K** [reply]
> I thought batch rename was builtin and I don't recall using any plugins

It *should* be builtin by default unless you compiled with `O_NOBATCH=1`.

**Reply by rieje** [reply]
Hmm, it seems batch rename fails when I'm running it inside tmux instance for some reason. If I run it outside tmux, I don't need `.nmv` explicitly installed and it works as expected. I did not edit `O_NOMATCH`.

**Reply by N-R-K** [reply]
> it seems batch rename fails when I'm running it inside tmux instance for some reason.

Can't reproduce here.
